solidus_frontend.gemspec

```diff
@@ -23,8 +33,9 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.5.0'
   s.required_rubygems_version = '>= 1.8.23'
 
-  s.add_dependency 'solidus_api', s.version
-  s.add_dependency 'solidus_core', s.version
+  s.add_dependency 'solidus_api', ">= #{s.version}"
+  s.add_dependency 'solidus_core', ">= #{s.version}"
+  s.add_dependency 'solidus_support', '~> 0.5'
```

Comment:
Is `solidus_support` needed?

Reply:
Hmmm... I got the specs and the sandbox to pass without `solidus_support`. I'll remove it. Todo.